### ISSUE TITLE
Kodi Built-in Functions 

### ISSUE THREAD

**Issue by uvjim** [albaintor/integration-kodi#3]
### Description

Is there any way to use the Kodi built-in functions (https://kodi.wiki/view/List_of_built-in_functions) in the integration?  I didn't spot one.  I have seen that using the remote entity achieves more of what I wanted than the media player entity did - I don't really have a need for the media player part on the R2 device.

Ideally what I was looking for, in the first instance, was to be able to create shortcuts on the remote (soft buttons) to access the Movie Sets, Movie Genre screens etc.  This would need me to use the `ActivateWindow` function passing the correct library path and the optional return parameter.  The keymap doesn't seem to have these defined.

**Comment by uvjim** [comment]
To answer my own question here... Looks like it'll need a wrapper around `call_method` from `pykodi`, which is already being done in some places. This generic wrapper would then need to be exposed higher up too and allow the remote UI to pass parameters in.

All this is much easier if we can predefine much of this to stop the need for parameter passing. I'll stick to using scripts in Home Assistant until I find the time to work it out better.

**Comment by albaintor** [comment]
This is not possible to define parameters at the integration level and make them available for user setting in the configurator. 
New commands are possible but with no parameters or static ones

**Comment by albaintor** [comment]
Now they're quite focused on R3 and the new UI so this is not the best timing. You can still reach them more easily on discord

**Comment by uvjim** [comment]
Just a quick follow-up on this one.  It appears that I can happily create my own keymap and have it loaded by Kodi.  The buttons used by this integration will then correspond to the newly loaded/merged keymap - not only the one linked to in the README.

I only found this when I replaced my Kodi device and settings were left at default, which changed the functionality of one of my buttons on the remote.

This means that I can now remove the scripts in Home Assistant that opened the Kodi Windows I was interested in.

Thanks again for this integration.  It's working really well for me and made the R2 become my daily driver.
